src/auth/oauth2client.ts

```diff
@@ -34,6 +34,19 @@ export enum CodeChallengeMethod {
 export interface GetTokenOptions {
   code: string;
   codeVerifier?: string;
+  /**
+   * The client ID for your application. The value passed into the constructor
+   * will be used if not provided. Must match any client_id option passed to
+   * a corresponding call to generateAuthUrl.
+   */
+  client_id?: string;
```

Comment:
Let's use `clientId` here and `redirectUri` below, since we're dealing with property names not bound to the API.

Reply:
Thanks Justin - I had that thought originally as well, but then went with `redirect_uri` and `client_id` for consistency with the opts properties currently passed to `generateAuthUrl()`, where they similarly appear as-written within the URL query params. If you think it's unlikely they're being passed to `generateAuthUrl()` elsewhere in the wild, I'd feel better about changing them in BOTH places (i.e. [https://github.com/google/google-auth-library-nodejs/blob/master/src/auth/oauth2client.ts#L415-L416](here) as well) for the same reason. Consistency across the two would allow the calling code to be a little cleaner.

That said, there's some risk -- I'd hazard a guess they're probably not in wide use -- but no data to back that up, and if they are, could be a breaking change for somebody. I did just go looking and found references to`redirect_uri` in the comments in a couple of places in google-api-nodejs-client, but no executable code.  

I saw this PR was reviewed but not yet merged - I'd be inclined to suggest it as-is with the above in mind, but if the naming change is still needed, happy to do it; please let me know what you think about doing so only here, or in both places.